File: c++ programming/arr.cpp
#include<iostream>
using namespace std;

int main()
{
   int a[4]={1,2,3,4};
   int j;

    cout<<a[0];
    cout<<a[1];
    cout<<a[2];
    cout<<a[3];
 
    cout<<"Your array is:"<<endl;

    for(j=0;j<4;j++)
    {
        cout<<a[j];
    }

    return 0;
}

File: Java/arr.java
import java.util.*;

public class arr {
    public static void main(String[] args) {
       int[][] a={{1,2,3},{2,3,4},{4,5,6}};


        System.out.println("Your array is:");
    try {
        for (int i = 0; i <= a.length; i++) {
            for (int j = 0; j <= a.length; j++) {
                System.out.println(a[i][j]);
            }
        }
    }catch (Exception e)
    {
        System.out.println(e);
    }
    }
}
